sys/linux/socket.txt

```diff
@@ -236,11 +236,15 @@ sockopt_so_timestamping = SOF_TIMESTAMPING_TX_HARDWARE, SOF_TIMESTAMPING_TX_SOFT
 
 # Since some socket ioctls are forwarded to the network device, adding device ioctls here as well.
 
-# http://lxr.free-electrons.com/source/include/uapi/linux/sockios.h
+# http://lxr.free-electrons.com/linux/latest/source/include/uapi/linux/sockios.h
 
-# http://lxr.free-electrons.com/source/include/uapi/asm-generic/sockios.h
+# http://lxr.free-electrons.com/linux/latest/source/include/uapi/asm-generic/sockios.h
 
-# TODO: add SIOCADDRT, SIOCDELRT, SIOCRTMSG, SIOCDRARP, SIOCGRARP, SIOCSRARP, SIOCGIFVLAN, SIOCSIFVLAN
```

Comment:
Added SIOCADDRT, SIOCDELRT, SIOCGIFVLAN, SIOCSIFVLAN and removed deprecated rarps(SIOCDRARP, SIOCGRARP, SIOCSRARP) as well as SIOCRTMSG that has only been used in inet ioctl, which is covered in socket_inet.txt.